installations/Podfile

```diff
@@ -0,0 +1,12 @@
+platform :ios, '9.0'
+use_frameworks!
+
+# Replace both pods by pod 'Firebase/Installations' once FirebaseInstallations is public.
```

Comment:
Change this into a TODO to use stable pod once M61 is released.

Reply:
Updated to a more specific TODO comment